templates/telco/index.html

```diff
@@ -298,20 +301,64 @@ <h2>Real-time kernel for telco</h2>
     </div>
   </section>
 
-  <section class="p-strip--suru-blog-header">
+  <section class="p-strip">
+    <div class="u-fixed-width">
+      <div class="p-card">
+        <div class="row">
+          <div class="col-4">
+            <a href="/kubernetes">
+              {{ image (
+              url="https://assets.ubuntu.com/v1/cee82ecd-Canonical Kubernetes logo.svg",
+              alt="Canonical Kubernetes",
+              width="187",
+              height="48",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+              }}
+            </a>
+          </div>
+          <div class="col-8">
+            <p class="p-heading--5">Infrastructure for cloud native telco</p>
+            <ul class="p-list">
+              <li class="p-list__item is-ticked">
+                Modular and scalable edge clouds with Canonical Kubernetes on MicroCloud or bare metal
+              </li>
+              <li class="p-list__item is-ticked">
+                Reliable and stable core clouds with Canonical Kubernetes on OpenStack or bare metal
+              </li>
+              <li class="p-list__item is-ticked">Repeatable and trusted deployment and operations for CNFs</li>
+              <li class="p-list__item is-ticked">Optimal and minimally-sized containers with chiselled Ubuntu images</li>
+            </ul>
+            <p>Learn how we support cloud-native telco with:</p>
+            <ul class="p-inline-list u-responsive-realign">
+              <li class="p-inline-list__item">
+                <a href="/blog/combining-distroless-and-ubuntu-chiselled-containers">Chiselled Ubuntu&nbsp;&rsaquo;</a>
+              </li>
+              <li class="p-inline-list__item">
+                <a href="/kubernetes">Canonical Kubernetes&nbsp;&rsaquo;</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip is-bordered">
```

Comment:
Change `p-strip is-bordered` with `p-strip--light`, it should work a bit better visually